vercel.json

```diff
@@ -310,6 +310,8 @@
         { "source": "/how_to/gui", "destination": "/divingDeeper/gui/gui" },
         { "source": "/How_To/Gui3D", "destination": "/divingDeeper/gui/gui3D" },
         { "source": "/how_to/gui3d", "destination": "/divingDeeper/gui/gui3D" },
+        { "source": "/How_To/MRTK", "destination": "/divingDeeper/gui/mrtk" },

```

Comment:
This file is only meant to keep back-compat linking correct. Was there ever an MRTK page that needs to be linked accordingly? If not you can remove this change.